### PR TITLE
Use Sentry for error reporting

### PR DESCRIPTION
**Pull request by tijmenb**
This commit makes the app use Sentry for error reporting. 

- Remove Airbrake gem & config
- Add [govuk_app_config][] which adds the Sentry gem ([sentry-raven][])
- Update slimmer to version 11.0.1, which removes Airbrake support

[govuk_app_config]: https://github.com/alphagov/govuk_app_config
[sentry-raven]: https://github.com/getsentry/raven-ruby

https://trello.com/c/pcht845E

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -306,8 +304,7 @@ DEPENDENCIES
   sass-rails (~> 5.0.3)
   simplecov (~> 0.10.0)
   simplecov-rcov (~> 0.2.3)
-  slimmer (~> 10.1.3)
-  statsd-ruby (= 1.3.0)
+  slimmer (~> 11.0.1)
```
Comment (whoojemaflip) [inline Gemfile.lock]: This seems unrelated to the rest of the PR?

Comment (tijmenb) [inline Gemfile.lock]: Updated the PR description.
